### DISCUSSION TITLE
How to send video or audio with document?

### DISCUSSION THREAD

**Discussion by Slavecode** [open-wa/wa-automate-nodejs#2636] [Q&A] [answered]
![20220407_070027](https://user-images.githubusercontent.com/43001614/162087016-b1590917-42fc-484a-8a8a-a9c5cc323cfd.jpg)
How to send video or audio with document?

**Comment by smashah** [comment, marked as answer]
See the docs:

https://docs.openwa.dev/classes/api_Client.Client.html#sendFile

Set `withoutPreview` to `true`:

<img width="814" alt="CleanShot 2022-04-08 at 02 28 42@2x" src="https://user-images.githubusercontent.com/5903821/162345417-6a017ca5-95c3-477d-9ebd-f3098422f317.png">

E.g

```js
await client.sendFile(chatId, "https://file-examples.com/storage/fe18180c4e624f64f9dba74/2017/04/file_example_MP4_480_1_5MG.mp4", "test.mp4", "test file caption", null, true, false, true)
```

**Reply by Slavecode** [reply]
Thx
